doc/source/admin/apache.md

```diff
@@ -221,16 +209,18 @@ previous section:
     ```
 
 2. The Galaxy application needs to be aware that it is running with a prefix (for generating URLs in dynamic pages).
-   This is accomplished by configuring uWSGI (the `uwsgi` section in `config/galaxy.yml`) like so and restarting Galaxy:
+   This is accomplished by configuring Galaxy in your `config/galaxy.yml` file like so and restarting Galaxy:
```

Comment:
```suggestion
   This is accomplished by configuring Galaxy and Gunicorn in your `config/galaxy.yml` file like so and restarting Galaxy:
```
Maybe "Galaxy and Gunicorn"? Otherwise, I think, it might be easy to miss the gravity config lines in this example.